wormhole-connect/src/utils/wallet/index.ts

```diff
@@ -339,6 +339,10 @@ const mapWallets = (
   skip: string[] = [],
 ): WalletData[] => {
   return Object.values(wallets)
+    .filter(
+      (wallet, index, self) =>
+        index === self.findIndex((o) => o.getName() === wallet.getName()),
```

Comment:
@sebastianscatularo I don't think I've fully understood this filter :-/
Are we filtering out the ones that doesn't match the name at the same index from the iterator?